### ISSUE TITLE
Manually created user groups do not display in Group column of Users table

### ISSUE THREAD

**Issue by erinysong** [cisagov/manage.get.gov#2662]
### Current Behavior

When a User is assigned a User Group that is not admin or analyst, the Users table does not accurately reflect the assigned user group.

### Expected Behavior

The Group column of the Users table should display all User Groups a User belongs to.

### Steps to Reproduce

1. In /admin, create a new User Group.
2. Go to the Users table and navigate to any User record. In Permissions > Groups, assign yourself your newly created group.
![image](https://github.com/user-attachments/assets/8faf5b24-0832-41f6-b3c2-a165a9930203)
3. Go back to the Users table and find the row for the User record you changed. The new assigned user group is not displayed in the Groups column.
![image](https://github.com/user-attachments/assets/4b235bd3-f794-4d74-9b39-e07e97542c69)

**Comment by abroddrick** [comment]
Yes @katypies it's a display issue in the groups column, also what is the reasoning that priority is "high"?

**Comment by abroddrick** [comment]
Hey pinging again @katypies, any thoughts on the above?
